website/static/games/ridgerider_games/minesweeper/minesweeper.css

```diff
@@ -1,29 +1,96 @@
 body {
-    background-color: green; 
+    background-color:green; 
 }
-#info, #settings {
+#info {
+    left: 20%;
+    position: fixed;
```

Comment:
Using flex to position items is usually best if you want your elements to work across different screen sizes instead of position fixed and having left/right/top/bottom values. Flex can be a little tough to use at first, but if you play around with it in a playground like Codepen it's easy to learn: https://codepen.io/team/css-tricks/pen/EKEYob

The below class is probably the most common CSS out there because it takes any div, centers it on the screen, and displays children elements in a row or column.

```css
.my-flex-class {
  display: flex;
  justify-content: center;
  align-items: center;
  /* row for child elements to be left and right. column for child elements to be up and down */
  flex-direction: row;
}
```

For more info on flexbox, this article is good  https://css-tricks.com/snippets/css/a-guide-to-flexbox/